test/kb_ea_utils_server_test.py

```diff
@@ -327,6 +327,87 @@ def getPairedEndLibInfo(self, read_lib_basename, lib_i=0):
         return new_obj_info
 
 
+    def getPairedEndInterleavedLibInfo(self, read_lib_basename, lib_i=0):
+        if hasattr(self.__class__, 'pairedEndLibInfo_list'):
+            try:
+                info = self.__class__.pairedEndLibInfo_list[lib_i]
+                name = self.__class__.pairedEndLibName_list[lib_i]
+                if info != None:
+                    if name != read_lib_basename:
+                        self.__class__.singleEndLibInfo_list[lib_i] = None
+                        self.__class__.singleEndLibName_list[lib_i] = None
+                    else:
+                        return info
+            except:
+                pass
```

Comment:
This all seems pretty complicated and is difficult to understand to a person coming at it fresh. I assume this method is staging data and memoizing it. If I were writing this now, I would just stage the data needed for a test at the beginning of the test or, if it was used by multiple tests, at the beginning of the test suite.

I don't think changing this is necessary for merging the PR.